cli/cmd/build/options.go

```diff
@@ -39,10 +39,6 @@ func init() {
 		"Categories to set for the agent. Overrides builder defaults. Example usage: --category category1 --category category2",
 	)
 
-	// Creation time (only for dev purposes)
-	flags.StringVar(&opts.CreatedAt, "created-at", "", "Agent creation time in RFC3339 format")
-	_ = flags.MarkHidden("created-at")
-
```

Comment:
this needs to stay for now as CSIT is using this flag to avoid creation time settings